### PR TITLE
add more tests

### PR DESCRIPTION
**Pull request by rpreen**
Currently adds (and passes tests):
* attack_utilities/
* tests/test_safedecisiontreeclassifier.py
* tests/test_saferandomforestclassifier.py

Omitted due to (missing) data requirement:
* tests/test_safesvc.py

Do we need to add data fetching to the CI runner to run such tests?

### CODE REVIEW

File: attack_utilities/estimator_template.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+'''
```
Comment (simonrnss) [inline attack_utilities/estimator_template.py:1]: I think this probably needs better commenting / docs.
Do we actually use it though? I thought these classes had been superseded by the safe wrappers?

File: attack_utilities/dp_svc_example.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+'''
```
Comment (simonrnss) [inline attack_utilities/dp_svc_example.py:1]: Not sure we need this code? And if we do, it would find a more natural home in the examples folder?

**Review by simonrnss — CHANGES_REQUESTED** [review]
A couple of suggestions -- not sure why we need the attack_utilities code? Where is it being imported?

**Comment by simonrnss** [comment]
@rpreen please also check pylint output -- if you're happy to overrule its variable name issues then adding
```python
# pylint: disable = invalid-name
```
to the offending files will shut pylint up. It's also finding near duplicate code in the test files. This might be unavoidable.

**Comment by rpreen** [comment]
Yeah, the `attack_utilities` should be removed if it's not needed. I think the pylint clean up should be a new issue/PR though since most of the complaints are about the safe model modules which aren't really dealt with in this PR (this is about adding tests)

**Comment by mccarthyajb** [comment]
OK, to be clear that's removing the entire attack_utilities directory? 
I don't think safesvc references it anymore.

**Comment by rpreen** [comment]
yes

**Comment by mccarthyajb** [comment]
OK, we were still importing GenericEstimator and the DPSVC class was inheriting it. 
I've removed those dependencies and the attack_utilities directory. The test are successful without them.

**Pull request merged**